internal/pkg/earpointer/analysis.go

```diff
@@ -592,6 +596,7 @@ func (vis *visitor) unifyCallWithContexts(arg ssa.Value, param ssa.Value, callsi
 
 // Handle calls to builtin functions: https://golang.org/pkg/builtin/.
 func (vis *visitor) visitBuiltin(builtin *ssa.Builtin, instr ssa.Instruction) {
+	// TODO: support more library functions (https://github.com/google/go-flow-levee/issues/312)
```

Comment:
```suggestion
	// TODO(#312): support more library functions
```

Suggesting, as this is our convention for referencing issues from TODOs.